.pre-commit-config.yaml

```diff
@@ -126,6 +126,8 @@ repos:
             "--disable=bad-continuation,cyclic-import",
             "--generated-members=torch.*,cv2.*",
             "--min-similarity-lines=15",
+            "--ignore-docstrings=yes",
```

Comment:
why changing the global pre-commit config?

Reply:
This is just a test to see if we can solve the duplicate-code problem.